_topic_map.yml

```diff
@@ -1360,6 +1360,24 @@ Topics:
     File: images-other-jenkins
   - Name: Jenkins agent
     File: images-other-jenkins-agent
+  - Name: Source-to-image
+    File: using-s21-images
+  #- Name: Source-to-image
```

Comment:
Moving these files to our unused folder. They will be available if we need them, but they will not build. 

Replaced by using-s21-images.adoc file, which addressed S2I images without going into a deep-dive for each. Based on @adambkaplan's feedback.

Reply:
Since these files are unused, I think they require little review.